assets/models/system/openai-whisper-large/spec.yaml

```diff
@@ -2,7 +2,7 @@ $schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
 name: openai-whisper-large
 path: ./
 properties:
-  SHA: e80f01dc6f24d14fdb05a0a3cfdb7f0ab2ec17fe
+  SHA: ""
```

Comment:
Could we update it to right value?

Reply:
@BhavanaTumma  Can you please let add you comments

Reply:
Since this was manually modified and updated, we don't have a SHA value, if we don't want to keep an empty tag, we can remove it